changelog/3770.breaking.1.rst

```diff
@@ -0,0 +1 @@
+``Fido.search`` will now return results from all clients which match a query, you no longer have to make the query specific to a single client. This means that searches involving the 'eve' and 'rhessi' instruments will now potentially also return results from the VSO. For `~sunpy.net.dataretriever.RHESSIClient` you can now specify ``a.Physobs("summary_lightcurve")`` to exclude VSO results.
```

Comment:
`instruments will now potentially` is the potentially due to the fact that it will return VSO results as well as the client results if they exist? maybe wording should be more along the lines of, `the search will also return results from vso if they exist`?